components/canvas/raqote_backend.rs

```diff
@@ -557,6 +560,29 @@ impl GenericDrawTarget for raqote::DrawTarget {
             &DrawOptions::Raqote(draw_options),
         );
     }
+    fn fill_text(
+        &mut self,
+        text: String,
+        x: f32,
+        y: f32,
+        pattern: canvas_data::Pattern,
+        draw_options: &DrawOptions,
+    ) {
+        let font = SystemSource::new()
```

Comment:
This should not sandbox-safe. It should be done in the chrome process, not the content process.